src/styles/main.scss

```diff
@@ -1,3 +1,14 @@
+@import './styles/variables';
+@import './styles/card';
+@import './styles/header';
+@import './styles/stars-review';
+
 body {
+  box-sizing: border-box;
   margin: 0;
 }
+
+p {
+  margin: 0;
+  padding: 0
+}
```

Comment:
```suggestion

```
Consider moving these styles to a different file to keep your main file clean and clear